static/app/views/ddm/utils/parseMetricWidgetsQueryParam.tsx

```diff
@@ -124,10 +136,20 @@ export function parseMetricWidgetsQueryParam(
         return null;
       }
 
+      const id = parseNumberParam(widget, 'id');
+      if (id === undefined || id < 0) {
```

Comment:
nit: i assume that we want this?
```suggestion
      if (id === undefined || id === NO_QUERY_ID) {
```